pathod/pathoc.py

```diff
@@ -73,7 +73,7 @@ def __str__(self):
             parts.append("\tPubkey: %s bit %s" % (pk.bits(), t))
             s = certutils.SSLCert(i)
             if s.altnames:
-                parts.append("\tSANs: %s" % " ".join(s.altnames))
+                parts.append("\tSANs: %s" % b" ".join(s.altnames))
```

Comment:
I printed out the parts. Here's how it looks currently:

```
Application Layer Protocol: b'http/1.1'
Cipher: AES256-GCM-SHA384, 256 bit, TLSv1/SSLv3
SSL certificate chain:
  Certificate [0]
        Subject: 
                b'CN'=b'pathod.net'
        Issuer: 
                b'CN'=b'mitmproxy'
                b'O'=b'mitmproxy'
        Version: 2
        Validity: b'20160611160814Z' - b'20190613160814Z'
        Serial: 14658340949850
        Algorithm: b'sha256WithRSAEncryption'
        Pubkey: 2048 bit RSA
        SANs: b'test1.com test2.com'
```

Weird mix of bytes & strings.